doc/vscode.md

```diff
@@ -0,0 +1,17 @@
+Integration with vscode
+-----------------------
+It's easy to produce output for parsing with the [coverage gutters](https://marketplace.visualstudio.com/items?itemName=ryanluker.vscode-coverage-gutters)
+extension to vscode. The `--cobertura-only` option is well-suited for this, which only outputs a
+`cov.xml` file and the coverage database in the output directory. Coverage information can then
+be directly shown in the "gutter" beside the editor, and is automatically updated by the
+extension when the coverage changes.
+
+To produce this output, just run
+
+```
```

Comment:
do not forget to add sh, yaml or else
it make it easier to look at

Reply:
Ah, right, thanks! Will fix with some later commit.